src/main/java/longah/node/Transaction.java

```diff
@@ -58,19 +92,35 @@ public Transaction(Member lender, ArrayList<Subtransaction> subtransactions,
      * @throws LongAhException If the user input is in an invalid format or value.
      */
     public void parseTransaction(String expression, MemberList members) throws LongAhException {
-        // User input format: [Lender] p/[Borrower1] a/[amount1] p/[Borrower2] a/[amount2] ...
+        // User input format: [Lender] t/[transactionTime(opt)] p/[Borrower1] a/[amount1] p/[Borrower2] a/[amount2] ...
```

Comment:
Was possible to keep the previous formatting by splitting the expression using t/ before the other established ways of processing, but I leave up to you if you want to change. Both are okay.